majava/basic.py

```diff
@@ -47,3 +51,32 @@ def __eq__(self, other):
 
     def __repr__(self):
         return ', '.join(it.__name__ for it in self.types)
+
+
+class AnyOf:
+    """Actual value must match at least one of the expected values."""
+
+    def __init__(self, value):
+        self.value = value
+
+    def __eq__(self, other):
```

Comment:
Instead of implementing its own `__eq__`, we need to inherite from `Matcher` and implement `_match(self, other)`. Check, how `DictContains` works.